src/routes/CreateSafePage/CreateSafePage.tsx

```diff
@@ -46,9 +46,11 @@ function CreateSafePage(): ReactElement {
   const provider = !!providerName && !isWrongNetwork
 
   useEffect(() => {
-    async function checkIfSafeIsPendingToBeCreated() {
+    const checkIfSafeIsPendingToBeCreated = async (): Promise<void> => {
       setIsLoading(true)
-      const safePendingToBeCreated = (await loadFromStorage(SAFE_PENDING_CREATION_STORAGE_KEY)) as CreateSafeFormValues
+      const safePendingToBeCreated = await Promise.resolve(
```

Comment:
Removing the promise completely here was breaking the CreateSafePage tests for a mysterious reason, so I kept this for now.

Reply:
Maybe add a `@TODO:` comment?